ui/elastic/src/App.js

```diff
@@ -89,9 +89,33 @@ const HitsTable = ({ hits }) => {
 
 const sortVersions = (filters) => {
   return filters.sort((a, b) => {
-    const majorA = parseInt(a.key.split(".", 1)[0], 10)
-    const majorB = parseInt(b.key.split(".", 1)[0], 10)
-    return majorB - majorA
+    const partsA = a.key.split(".")
+    const partsB = b.key.split(".")
+    if (partsA[0] !== partsB[0]) {
+      // sort on major version.
+      return parseInt(partsB[0], 10) - parseInt(partsA[0], 10)
+    }
+    if (partsA[1] !== partsB[1]) {
+      const subPartRegex = /^(\d+)([a-zA-Z]+)?(\d+)?([a-zA-Z]+)?/ // Eg: 0b12pre
```

Comment:
It seems like this enforces that `0b12pre` can only happen in after the first period and not after a second, but you also report seeing `38.0.5b1-2`. Is this OK?